demisto_sdk/commands/validate/tools.py

```diff
@@ -172,3 +177,51 @@ def is_indicator_pb(playbook: Playbook):
         (i.get("playbookInputQuery") or {}).get("queryEntity") == "indicators"
         for i in playbook.data.get("inputs", {})
     )
+
+
+def extract_rn_headers(rn_content) -> Dict[str, List[str]]:
+    """
+        Extracts the headers from the release notes file.
+    Args:
+        None.
+    Return:
+        A dictionary representation of the release notes file that maps content types' headers to their corresponding content items' headers.
+    """
+    headers: Dict = {}
+    # Get all sections from the release notes using regex
+    rn_sections = CONTENT_TYPE_SECTION_REGEX.findall(rn_content)
+    for section in rn_sections:
+        section = filter_none_values(ls=section)
+        content_type = section[0]
+        content_type_sections_str = section[1]
+        content_type_sections_ls = CONTENT_ITEM_SECTION_REGEX.findall(
+            content_type_sections_str
+        )
+        if not content_type_sections_ls:
+            #  Did not find content items headers under content type - might be due to invalid format.
+            #  Will raise error in rn_valid_header_format.
+            headers[content_type] = []
+        for content_type_section in content_type_sections_ls:
+            content_type_section = filter_none_values(ls=content_type_section)
+            if content_type_section:
+                header = content_type_section[0]
+                if headers.get(content_type):
+                    headers[content_type].append(header)
+                else:
+                    headers[content_type] = [header]
+    return headers
+
+
+def filter_rn_headers(headers: Dict) -> None:
```

Comment:
not a great name for a function, doesnt really define what it filters out, lets think of a better name